unlock-app/src/components/content/event/Layout/EventsLayoutEnum.tsx

```diff
@@ -0,0 +1,6 @@
+enum EventsLayoutEnum {
```

Comment:
Nit: no need to specify the "type" of a variable in its name.
```suggestion
enum EventsLayout {
```